Test members

ANKerD opened the pull request.

wassoares commented on src/test/java/cloud/fogbow/rcs/core/service/CatalogServiceTest.java:
```diff
@@ -0,0 +1,117 @@
+        PowerMockito.when(MembershipServiceResponse.fromJson(Mockito.anyString())).thenReturn(Mockito.mock(MembershipServiceResponse.class));
+
+        BDDMockito.given(HttpRequestClient.doGenericRequest(Mockito.any(), Mockito.any(), Mockito.any(),
+                Mockito.any(HashMap.class))).willReturn(httpResponse);
```
To make the code more uniform, remove the **HashMap.class** parameter or use check values ​​with _"Mockito.eq()"_ on all parameters. Fix this for other occurrences.

ANKerD replied:
Okay.

wassoares commented on src/test/java/cloud/fogbow/rcs/core/service/CatalogServiceTest.java:
```diff
@@ -0,0 +1,117 @@
+
+        // exercise
+        catalogService.requestMembers();
+        Assert.fail();
```
Check using **try/catch** instead of **expected**, and use _"assertEquals()"_ checking the message thrown on the exception.

ANKerD replied:
Done.

wassoares commented on src/test/java/cloud/fogbow/rcs/core/service/CatalogServiceTest.java:
```diff
@@ -0,0 +1,117 @@
+        Mockito.doReturn(new ArrayList<>()).when(catalogService).listProviderMembers(Mockito.any());
+
+        // exercise
+        catalogService.requestMembers();
```
Use **"this"** on all calls of the **catalogService** attribute. Fix this for other occurrences.

ANKerD replied:
Fixed.

ANKerD commented:
About those changes on pom.xml, I added it because ras had a problem with some dependencies imported by some libs. I added those exclusions just to prevent this. Do you want me to remove?
